distribution/docs/src/main/resources/_contents/_configuring/configuring-from-the-admin-console-contents.adoc

```diff
@@ -0,0 +1,156 @@
+
+The ${branding} configuration can be performed through the ${admin-console} User Interface.
+
+==== Accessing the ${admin-console}
+
+Access the ${admin-console} from a web browser to configure the system.
+
+* Navigate to the ${admin-console}. (Default: \${secure_url}/admin)
+* Enter Username and Password. (Default: `admin/admin`)
+
+==== Initial Configuration
+
+The first time the ${branding} ${admin-console} runs, the initial configuration steps appear.
+
+. Click *Start* to begin.
+. On the next screen, general configuration settings such as host address, port and site name can all be configured. (See <<Configuring ${branding} Global Settings>> for important settings to configure)
+. Next, choose between a standard installation and a full installation. (Individual applications can be added, removed or deactivated later)
```

Comment:
Full installation doesn't exist anymore.
Also, since the installation profiles can be configured based on the distribution (branding), maybe we should only name the standard one and mention that their could be others based on the distribution used. Thoughts?